internal/k8s/app_protect_resources.go

```diff
@@ -55,14 +55,22 @@ func ValidateAppProtectLogConf(logConf *unstructured.Unstructured) error {
 	return nil
 }
 
-var logDstEx = regexp.MustCompile(`syslog:server=((?:\d{1,3}\.){3}\d{1,3}|localhost):\d{1,5}`)
+var logDstEx = regexp.MustCompile(`(?:syslog:server=((?:\d{1,3}\.){3}\d{1,3}|localhost):\d{1,5})|stderr|(?:\/[\S]+)+`)
+var logDstFileEx = regexp.MustCompile(`(?:\/[\S]+)+`)
 
 // ValidateAppProtectLogDestinationAnnotation validates annotation for log destination configuration
 func ValidateAppProtectLogDestinationAnnotation(dstAntn string) error {
```

Comment:
Could you possibly add a unit test for this function?  It be great to test for each of the possible types of valid values as well as invalid values (a case per each`return  fmt.Errorf`)

Reply:
added tests, also removed one return fmt.Errorf since i found it is not reachable.